### PR TITLE
fix(git): ignore some compiled products generated by the kernel

### PR DESCRIPTION
**Pull request by sora1874**
git status will show：
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -47,6 +47,16 @@
 *.tar
 *.xz
 *.zst
+*.cmd
+*.d
+*.S
+*.config
```
Comment (amazingfate) [inline .gitignore:53]: There are files tracked by git.

**Review by amazingfate — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (amazingfate) [inline .gitignore:53]: there are files `*d`, `*.S` and `*.config` tracked by git.

Comment (sora1874) [reply .gitignore:53]: image
is not use .gitignore file ?

Comment (amazingfate) [reply .gitignore:53]: test.d is a dir: https://github.com/armbian/linux-rockchip/tree/rk-5.10-rkr5.1/tools/testing/selftests/ftrace/test.d
There are already *.S in git: https://github.com/armbian/linux-rockchip/blob/rk-5.10-rkr5.1/usr/initramfs_data.S
Also *.config: https://github.com/armbian/linux-rockchip/blob/rk-5.10-rkr5.1/tools/testing/selftests/wireguard/qemu/kernel.config

**Pull request closed without merging**